pegasus/sites.v3/code.org/public/educate/resources/videos.md

```diff
@@ -99,6 +99,14 @@ We are building a growing library of educational videos available for re-use by
 
 </div>
 
+<div class="col-50" style="float: left; padding: 10px;">
+
+<% facebook = {:u=>'https://youtu.be/LVV_93mBfSU'} %>
+<% twitter = {:url=>'https://youtu.be/LVV_93mBfSU', :related=>'codeorg', :text=>'THE INTERNET: How Search Works @codeorg'} %>
```

Comment:
Yeah, I'm not sure who chose this but I kept it for consistency. I think it helps it to stand out more on the page. Too much?